style.css

```diff
@@ -443,7 +443,175 @@ textarea {
+  left: 50%;
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.btn-close-pop-up {
+  position: absolute;
+  top: 0%;
+  right: 5%;
+  background: none;
+}
+
+.work-image-details {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.work-image-details > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+  content: url("./img/background/bk.png");
+}
+
+.work-title-mobile {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  color: #172b4d;
+  text-align: left;
+}
+
+.work-tags-mobile {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.tag-mobile {
+  background: #ebf0ee;
+  color: #3a4a42;
+  padding: 0.5em 0.75em 0.5em;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+}
+
+.work-discription-mobile {
+  color: #344563;
+  text-align: left;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  line-height: 1.5em;
+  padding-bottom: 15px;
+}
+
+.work-button-mobile {
+  display: flex;
+  justify-content: space-between;
+}
+
+.work-detail-btn-mobile {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  width: fit-content;
+  align-self: center;
+  background: #0e5a3b;
+  font-weight: 500;
+  font-size: 16px;
+  letter-spacing: 2px;
+  padding: 1em 1.5em;
+  color: white;
+  border-radius: 3px;
+  text-decoration: none;
+}
+
+.work-detail-btn-mobile > img {
+  margin-left: 1.3em;
+}
+
 @media (min-width: 768px) {
+  /** pop up css **/
+  .tag-mobile {
+    background: #ebf0ee;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 0.75em;
+    line-height: 1em;
+    letter-spacing: 0.03em;
+    color: #344563;
+    border-radius: 0.5em;
+  }
+
+  .my-works-details {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 60px;
+    grid-template-rows: 60px 50% 1fr 60px 1fr;
+    width: 85%;
+    margin-top: 30px;
+  }
+
+  .btn-close-details {
+    background: #ebecf0;
+    position: relative;
+    right: 0;
+    left: 0;
+    grid-column: -2 / -1;
+    grid-row: 1/2;
+  }
+
+  .work-image-details {
+    grid-column: 1 / -1;
+    grid-row: 2 / 3;
+  }
+
+  .work-image-details > img {
+    overflow: hidden;
+    box-sizing: border-box;
+    border-radius: 16px;
+    content: url("./img/background/Snapshoot-Portfolio-desktop.svg");
+  }
+
+  .work-title-mobile {
+    grid-column: 1 / 3;
+    grid-row: 3 / 4;
+    font-size: 1.5em;
+  }
+
+  .work-button-mobile {
+    grid-column: 3 / 5;
+    grid-row: 3 / 4;
+  }
+
+  .work-tags-mobile {
+    grid-column: 1 /-1;
+    grid-row: 4 / 5;
+  }
+
+  .work-discription-mobile {
+    grid-column: 1 / -1;
+    grid-row: 5 / 6;
+  }
+
```

Comment:
- [ ] Congratulations for making the mobile and desktop popup size fit properly :+1:, but as a project requirement you should stick as much as possible to the Figma template, so kindly set your popup in the middle of your page.

- [ ] As requested before you should make your title a little bigger :+1: 

- [ ] Also, the background should be blurred to make the user not see other elements but only the popup :+1:. [Here](https://youtu.be/wiIbol5tZaI) you can find a good video to make it possible.

**Your design**

![Screenshot from 2023-02-03 06-15-41](https://user-images.githubusercontent.com/105117832/216592524-a6113936-4a9a-4a9f-8fe9-b8f473fe74a8.png)

**Figma**

![Screenshot from 2023-02-03 04-52-56](https://user-images.githubusercontent.com/105117832/216592613-c3229a45-a88a-4f9b-8aed-9dbaff299d21.png)